anonymisations.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"crypto/sha1"
```

Comment:
Might be an idea to run `gofmt` over these files so we can see if there was anything we didn't do in the "Go" way: https://blog.golang.org/go-fmt-your-code (it may very well be a no-op!)

Reply:
As commented in https://github.com/intenthq/anon/pull/1#issuecomment-389856037, I vscode did it for me :)